docs/topics/tasks.txt

```diff
@@ -15,10 +15,12 @@ to be run elsewhere, potentially at a later date. This keeps requests fast,
 reduces latency, and improves the user experience. For example, a user
 shouldn't have to wait for an email to send before their page finishes loading.
 
-Django's new Tasks framework makes it easy to define and enqueue such work. It
+Django's Tasks framework makes it easy to define and enqueue such work. It
 does not provide a worker mechanism to run Tasks. The actual execution must be
 handled by infrastructure outside Django, such as a separate process or
-service.
+service. Given that, you will want to choose a
+:ref:`task backend <configuring-a-task-backend>` capable of executing tasks on
+that service.
```

Comment:
In my head (without me referring to the docs style guide mind you so if this breaks that then I'm fine with it), I was thinking a short info or warning call out box here stating that you will need a third-party package to execute tasks in the background.